libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1210,17 +1226,16 @@ def test_deployment(self, sns_data, sns_parity,
                 if not resp[0]:
                     LOGGER.debug("Did not get expected response: %s", resp)
                 ext_ip = resp[1]
-                port = resp[3]
+                port = resp[2]
```

Comment:
with resp[2] we would need to chnage the endpoint as well to https://<ip>:<port>, is that taken care?